pkg/apis/flyteworkflow/v1alpha1/workflow.go

```diff
@@ -298,3 +305,10 @@ type FlyteWorkflowList struct {
 	metav1.ListMeta `json:"metadata"`
 	Items           []FlyteWorkflow `json:"items"`
 }
+
+// This contains workflow-execution specifications and overrides.
+type ExecutionConfig struct {
+	// Maps individual task types to their alternate (non-default) plugin handlers by name.
+	TaskPluginImpls        map[string][]string
+	TaskPluginOverrideMode admin.PluginOverride_MissingPluginBehavior
```

Comment:
This is per task type, right? the way it's modeled in admin I mean...

Shouldn't the datastructure be similar here?
```
TaskPluginImpls.  map[string]TaskPluginOverride


type TaskPluginOverride struct {
[]string
missingpluginbehavior
}
```

Reply:
done, thanks @EngHabu

Reply:
hmmm Am I looking at an old sha? I don't see this changed...

Reply:
https://github.com/lyft/flytepropeller/pull/184/files#diff-79c4ac3e62dcfb7f9b67360a573a9b71R312 ?